source/quick-start.txt

```diff
@@ -7,193 +7,182 @@ Quick Start
+and manage your data. Go through the
+:atlas:`Get Started with Atlas <getting-started>` guide to set up a new
+Atlas account, free tier cluster (MongoDB instance), load datasets, and
+interact with the data.
 
-Next, create the ``.env`` file and add a variable called
-``MONGODB_URI``:
-
-.. code-block:: sh
-
-  touch .env
-  echo "MONGODB_URI=<connection_uri>" >> .env
-
-Add your Atlas Connection String
---------------------------------
-
-Open your newly created ``.env`` file. If you are unable to locate this
-file in your project directory, you may need to set your system to
-display hidden files. Once your ``.env`` is open, replace
-``<connection_uri>`` with your connection string from Atlas. Ensure you
-update ``<username>`` and ``<password>`` in your connection string.
+After completing the steps in the Atlas guide, you should have a new MongoDB
+cluster deployed in Atlas, a new database user, and sample datasets loaded
+into your cluster.
 
 Connect to your Cluster
 -----------------------
 
-For the final step, we'll write some code that tests your environment's
-connection to your instance of MongoDB.
-
-Create a file in your project directory called ``index.js``:
+In this step, we create and run an application that uses the Node.js MongoDB
+driver to connect to your instance of MongoDB and run a query on the sample
+data.
 
-.. code-block:: sh
+To connect to Atlas using the Node.js driver, we provide instructions to the
+driver on where and how to connect to your instance in a string called the
+*connection string*.
 
-  touch index.js
+Follow the
+:atlas:`Connect via Driver <driver-connection>` guide for instructions on
+how to set up a user and retrieve your connection string.
 
-Open this file and paste in the following code:
+Once you have your connection string, create a file called ``index.js`` in
+your ``node_quickstart`` directory. Add the following code, replacing the
+``uri`` variable with your connection string.
 
 .. code-block:: js
 
   const { MongoClient } = require("mongodb");
-  require("dotenv").config();
 
-  if (!process.env.MONGODB_URI) {
-    throw Error("MONGODB_URI not defined in a .env file. Did you set it up?");
-  }
+  // Replace the uri string with your MongoDB deployment's connection string.
+  const uri =
+    "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority&useUnifiedTopology=true";
 
-  const client = new MongoClient(process.env.MONGODB_URI, {
-    useUnifiedTopology: true
-  });
+  const client = new MongoClient(uri);
 
   async function run() {
     try {
-      // Connect the client to the server
       await client.connect();
-      // Establish and verify connection
-      await client.db("admin").command({ ping: 1 });
-      console.log("Successfully connected to Atlas.");
-      // your code after this comment
 
-      // your code before this comment
+      const database = client.db('sample_mflix');
```

Comment:
I like this code! How do we let users know that this code may fail / not match the expected output if they did not load a sample dataset?

Reply:
How about this sentence after the sample output:
"If or receive an error, check whether you included the proper connection string in the application and loaded the sample dataset."

Reply:
Excellent